sys/shell/commands/sc_cryptoauthlib.c

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_shell_commands
+ * @{
+ *
+ * @file
+ * @brief       Shell commands for the cryptoauthlib module
+ *
+ * @author      Lena Boeckmann <lena.boeckmann@haw-hamburg.de>
+ *
+ * @}
+ */
+#include <stdio.h>
+#include <stdint.h>
+#include "cryptoauthlib.h"
+
+void get_bin(char *result, uint8_t byte)
+{
+    for (int i = 0; i < 8; i++) {
+        result[i] = (byte << i & 0x80 ? '1' : '0');
+    }
+    result[8] = '\0';
+}
+
+int _read_config(int argc, char **argv)
+{
+    (void)argc;
+    (void)argv;
+    uint8_t data[128];
+    uint8_t data_count = 0;
+    char binary[9];
+
+    atcab_read_config_zone(data);
```

Comment:
on a second thought and a look at the lib API; maybe add some error handling here, i.e.,

```
if (atcab_read_config_zone(data) != ATCA_SUCCESS) {
    printf("Error reading config zone!\n");
    return 1;
}
```